src/components/base/accordion/Accordion.module.css

```diff
@@ -1,41 +1,48 @@
 .element {
-  border-bottom: 1px solid var(--iron);
+  display: flex;
+  flex-direction: column;
+  height: fit-content;
+}
+
+.wrapper:focus,
+.wrapper:hover {
+  z-index: 2;
 }
 
-.element:first-child {
-  border-top: 1px solid var(--iron);
+.wrapper {
+  composes: underlineContainer__only_internal_animations from "/src/components/base/animation/animations.module.css";
```

Comment:
sådan en der ved jeg ikke hvad er

Reply:
Det er css modules måde at extende en class på.

Og det kan ikke løses på andre måder akri? Havde du ikke lavet nogle class som kunne importeres osv?

Reply:
Det kan sagtens løses på andre måder. Jeg importerer bare klassen i stedet for, så :relaxed: